commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/util/DebugUtils.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.util;
+
+import org.eclipse.che.commons.annotation.Nullable;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Provides debug id for the class that implements {@link IDProvider}
+ *
+ * @author Vlad Zhukovskiy
+ */
+public final class DebugUtils {
```

Comment:
Because I saw many pieces of code that perform some bycicles...

Its a proposal to unify all this work. 

note: notification - its not a gwt component its a simple data object
```
        SimplePanel iconWrapper = new SimplePanel();
        iconWrapper.ensureDebugId(DebugUtils.getDebugId("some-prefix", notification));
        return iconWrapper;
```